recipes/flann/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "1.8.4" %}
+
+package:
+    name: flann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: 40d63fc7db5d1f6dd67364f24e4579aedb57e7b6
+
+build:
+    number: 0
+    string: {{PKG_BUILDNUM}}_g{{GIT_FULL_HASH[:7]}}
```

Comment:
Assuming this is for development versions and we come to some consensus on this version and all of that, I'd like us to try to do something like versioneer here. I have some code that I used for this that is admittedly pretty disgusting, but I think this is a good place to start. This would replace the version string. Then the build string would be unnecessary.

    {{ environ.get('GIT_DESCRIBE_TAG', '')[1:] }}{{ '+' + environ.get('GIT_DESCRIBE_NUMBER', '0') + '.' + environ.get('GIT_DESCRIBE_HASH', '0') if environ.get('GIT_DESCRIBE_NUMBER', '0') != '0' else "" }}